src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(Color color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double calculateArea() {
+        return Math.PI * Math.pow(radius, 2);
+    }
+
+    @Override
+    public void drawFigure() {
+        System.out.println("Figure: circle, area: " + calculateArea() + " sq. units, radius: "
+                + getRadius() + " units, color: " + getColor());
```

Comment:
Instead of directly using `System.out.println` in `drawFigure()`, consider using an interface that abstracts the drawing mechanism, which would allow for different implementations of drawing (e.g., to a file, a GUI, etc.).